docs/pubsub-api.rst

```diff
@@ -0,0 +1,255 @@
+``gcloud.pubsub`` API
+=====================
+
+Connection / Authorization
+--------------------------
+
+- Inferred defaults used to create connection if none configured explicitly:
```

Comment:
Do we have an example of explicitly specifying credentials?

What about connecting to two different projects with two different sets of credentials?

Something like... 

```python
from gcloud import pubsub
project1 = pubsub.Connection(project_id='...', credentials_json='/path/to/key1.json')
topic = project1.create_topic('...')

project2 = pubsub.Connection(project_id='...', credentials_json='/path/to/key2.json')
topic = project2.create_topic('...')
```

..?